src/content/chapters/03_word_types.mdx

```diff
@@ -0,0 +1,181 @@
+the letter h. Note that if there is only one vhowel, then it cannot be h, as h
+must be between vowels.
+
+The use of "vhowel" is here to make explanations easier, it'll only be seen in
+this chapter.
+
+## Particles
+
+Particles facilitate forming an Eberban sentence. With them, you can express
+your thoughts with precision and use your own speaking style.
+
+Particles are grouped into families that share the same pattern. Each family is
+written in all caps using the alphabetically-first particle in its family. For
+example, `MI` represents `mi`, `mie`, `mio`, `me`, `ma`, etc... as the vowel
+order is "ieaou".
+
+Particles begin with a single non-sonorant consonant and are followed only by
+vhowels.
+
+Particles function in one of two ways. Some particles, like the `MI` family, act
+like predicates. `mi` is the property "_ is the singular first
+person/speaker/author", e.g. "I". Other particles, like `PA`, serve a
+grammatical purpose. `pa` begins an assertive sentence—that which the speaker
+asserts to be true.
+
+## Content Words
+
+Content words express the meaning of an Eberban sentence. All content words are
+predicates.
+
+The structure of content words may include two consonants in a row, but never
+more. These consonant pairs are categorised below:
+
+![A table that distinguishes consonant pairs. There are three types: initial, medial, and invalid pairs.](/consonant-pairs.png "All consonant pairs.")
+
+Legend:
+
+- <span class="text-[#b4c7dc] font-bold dark:drop-shadow-md">Initial pairs</span>
+- <span class="text-[#bf819e] font-bold dark:drop-shadow-md">Medial pairs</span>
+- <span  class="text-[#333333] font-bold dark:drop-shadow-md dark:[text-shadow:1px_1px_gray]">Invalid pairs</span>
+
+Just like the letter h, a medial pair can only appear between two vowels.
+
+### Roots
+
+Roots are the building blocks of the language and express meanings that would be
+too hard or long to convey otherwise. 
+
+There are two options for a root's structure.
+
+**Root Structure Option 1:**
+
+1. Begin the root with a non-sonorant consonant.
+1. Follow it with at least one vhowel.
+1. Two options:
+   1. Either follow it with a sonorant and any number of vhowels.
+   1. Or follow it with a medial pair and at least one vhowel.
+1. Optionally repeat step 3 as much as you'd like.
+1. End.
+
+Examples:
+
+- ber: property for "_ is Eberbanic/reflects Eberbanic culture".
+- ban: property for "_ is a language".
+- TODO INSERT TRANSITIVE:
+- TODO INSERT TRANSITIVE:
+
+**Root Structure Option 2:**
+
+1. Begin the root with an initial pair.
+1. End it with at least one vhowel.
+1. Optionally:
+   1. Either end it with a sonorant, and optionally go back to step 2.
+   1. Or follow it with a medial consonant pair and go back to step 2.
+
+Examples:
+
+- pcien: property for "_ is bread".
+- ckual: property for "_ is a planet".
+- TODO INSERT TRANSITIVE: 
+- gli: relation for "_ is happy that _ is true".
+
+### Compounds
```

Comment:
You should explain borrowings first, then tell what can appear inside compounds (root, particles, and a borrowing in first position only)